webauthn.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.4
 name: webauthn
-version: 0.5.0.1
+version: 0.5.0.2
```

Comment:
The version should increase to 0.6.0.0 since this is a breaking change to the API.